dev/integration_tests/android_semantics_testing/lib/main.dart

```diff
@@ -5,12 +5,14 @@
 import 'dart:async';
 import 'dart:convert';
 
+import 'package:android_semantics_testing/src/tests/popup_constants.dart';
```

Comment:
This should probably be imported via relative URL?

Reply:
Yeah, stupid IntelliJ keeps auto-importing things for me, even though I've turned off all the auto-import preferences I can find.

Fixed.